### DISCUSSION TITLE
SHA256 check for v0.48.0

### DISCUSSION THREAD

**Discussion by henrythebuilder** [starship/starship#2059] [Q&A] [answered]
Hello, thank you for everyone who created and contributes to starship.
I'have a problem with the last release, it seam that the checksum fail for [starship-x86_64-unknown-linux-gnu.tar.gz](https://github.com/starship/starship/releases/download/v0.48.0/starship-x86_64-unknown-linux-gnu.tar.gz) package.
In my system sha256 check sum did not match:

`bin/pkg/starship/release-0.48.0 `
`> cat starship-x86_64-unknown-linux-gnu.tar.gz.sha256 `
`305e1ff05e81319b58b03b4ddfdb99a2da0924cd670d623897d314a336f575be`

`bin/pkg/starship/release-0.48.0 `
`> sha256sum starship-x86_64-unknown-linux-gnu.tar.gz`
`ed80a9984ed4575042b8eade585058d6efadc1130bbc5f1af6b9accf9260a900  starship-x86_64-unknown-linux-gnu.tar.gz`

any help is appreciated
thanks in advance
Enrico

Reference file:
[starship-x86_64-unknown-linux-gnu.tar.gz](https://github.com/starship/starship/releases/download/v0.48.0/starship-x86_64-unknown-linux-gnu.tar.gz)
[starship-x86_64-unknown-linux-gnu.tar.gz.sha256](https://github.com/starship/starship/releases/download/v0.48.0/starship-x86_64-unknown-linux-gnu.tar.gz.sha256)

**Comment by chipbuster** [comment, marked as answer]
@henrythebuilder, thanks for catching this!

Due to an issue in GH Actions CI, the linux-gnu binary/hashes for the v0.48.0 release weren't autogenerated like they usually are, but manually compiled and built on my machine. It looks like I was moving a little too fast and accidentally SHA-summed the binary executable instead of the tar file--if you untar the file and hash the binary, you should get the old sum.

I've updated the release with the correct hash for the tarball. Thanks for letting us know!

**Comment by henrythebuilder** [comment]
Thanks for your work :+1:
